news/2023/2023-05-01-vocaloid-mapping-contest.md

```diff
@@ -0,0 +1,113 @@
+
+## Rules
+
+Believe it or not, contests have rules to keep everything fair.
+
+- **Submissions must be for the osu! ruleset only.** Comparing beatmaps of different modes in a judging scenario is unfortunately not realistic.
+- **Submissions must use one of the [selected songs](#songs).**
+- **Submissions must be in `.osz` format.**
+- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
+- **Submissions must not be publicly revealed until after the results are announced.**
+- **Excluding metadata, submissions must not include anything that could reveal your identity.** Metadata will be anonymised before submissions are relayed to judges.
+- **Collaborations are not allowed**. This is a solo contest.
+- **Submissions must not include storyboards or videos.**
+- **Your submission must not have any edits to the .osu that would not be possible through the editor.** You are permitted to change any difficulty settings, backgrounds, green lines, SV, hitsounds, etc.
+- **Your submission must have only one difficulty.**
+
+## Songs
+
+With 5 songs to choose from, we're confident that at least one will appeal to you!
+
+### MIMI - Moment (3:34 / 126 BPM)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/77/osz/MIMI%20-%20Moment.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/77/osz/MIMI%20-%20Moment.mp3" type="audio/mpeg">
+</audio>
+
+### Omoi - Ohne Herz (3:26 / 210 BPM)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/104/osz/Omoi%20-%20Ohne%20Herz.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/104/osz/Omoi%20-%20Ohne%20Herz.mp3" type="audio/mpeg">
+</audio>
+
+### MuryokuP - My Savior (2:49 / 190 BPM)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/120/Unfinished%20Eden/MuryokuP%20-%20My%20Savior.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/120/Unfinished%20Eden/MuryokuP%20-%20My%20Savior.mp3" type="audio/mpeg">
+</audio>
+
+### Yunosuke - Ziqqurat (3:52 / 163 BPM)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/244/Ziqqurat/Yunosuke%20-%20Ziqqurat.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/244/Ziqqurat/Yunosuke%20-%20Ziqqurat.mp3" type="audio/mpeg">
+</audio>
+
+### Rohi - Hitotsu no Chikai yo (2:35 / 130 BPM)
+
+[Download the beatmap template here!](https://assets.ppy.sh/artists/82/osz/Rohi%20-%20Hitotsu%20no%20Chikai%20yo,%20Eien%20no%20Uta%20to%20Nare.osz)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/82/osz/Rohi%20-%20Hitotsu%20no%20Chikai%20yo,%20Eien%20no%20Uta%20to%20Nare.mp3" type="audio/mpeg">
+</audio>
+
+---
+
```

Comment:
this one is not needed since there's a heading right below, and it'd also look bad.

```suggestion
```